data-serving/scripts/export-data/transform.py

```diff
@@ -84,10 +86,12 @@ def convert_event(event: dict[str, Any]) -> dict[str, Any]:
     suffix = event["name"]
     col_name = f"events.{suffix}"
 
-    return {
-        f"{col_name}.value": event.get("value"),
+    o = {
```

Comment:
OK to add meaningful variable names? It makes code easier to read and work with.